service/dependency.go

```diff
@@ -58,40 +57,13 @@ func extractVolumes(service *Service, dependency *Dependency, details dependency
 	return
 }
 
-func getDockerService(namespace string, dependencyName string) (dockerService swarm.Service, err error) {
-	ctx := context.Background()
-	cli, err := dockerCli()
-	if err != nil {
-		return
-	}
-	dockerServices, err := cli.ListServices(docker.ListServicesOptions{
-		Filters: map[string][]string{
-			"name": []string{strings.Join([]string{namespace, dependencyName}, "_")},
-		},
-		Context: ctx,
-	})
-	if err != nil {
-		return
-	}
-	dockerService = dockerServiceMatch(dockerServices, namespace, dependencyName)
-	return
-}
-
 func dependencyStatus(namespace string, dependencyName string) (status StatusType) {
-	ctx := context.Background()
-	cli, err := dockerCli()
+	dockerStatus, err := container.ServiceStatus([]string{namespace, dependencyName})
 	if err != nil {
-		return
+		panic(err)
```

Comment:
this panic shouldn't exists here, I guess we should return an error for this function or have a default behavior like before where it returns `STOPPED` status is an `int` so the initial value is 0 so `STOPPED` so if it returns it will return stop.

Maybe we should instead return the error but anyway have a panic here is not a viable solution I think

Reply:
I'm completely ok with you, I would prefer to return the error. I didn't to preserve the API and make the modification in a other PR.